README.md

```diff
@@ -80,6 +80,6 @@ for a pod with `RestartPolicy` set to `Never` or `OnFailure`. The so called run-
 
 1. Checking the `.spec.activeDeadlineSeconds` field was set to 3600:
    ```sh
-   $ oc get pods -n test -o json | jq '.items[0].spec.activeDeadlineSeconds'
+   $ oc get pods -n test -o json | jq '.spec.activeDeadlineSeconds'
```

Comment:
Please correct me if I'm wrong but we'll get multiple pods from `oc get pods` and we need to get first pod's activeDeadlineSeconds via `items[0]`. Have you tried this?

Reply:
@ardaguclu yes, if there are multiple pods then what is already here will work but since our readme has just one example pod i thought this might be apt. But if you think existing one should suffice, we can close this, no issues for me.

Reply:
`oc get pods -n test -o json` gives a list of pods even though there's only a single pod in a namespace.
```sh
$ oc get pods
NAME      READY   STATUS    RESTARTS   AGE
example   1/1     Running   0          76s
$ oc get pods -o json | head -6
{
    "apiVersion": "v1",
    "items": [
        {
            "apiVersion": "v1",
            "kind": "Pod",
            ...
}
```

Reply:
Running the updated command gives me
```sh
$ oc get pods -n test -o json | jq '.spec.activeDeadlineSeconds'
null
```